### ISSUE TITLE
Also pack the database

### ISSUE THREAD

**Issue by giovapanasiti** [wojtkowiak/meteor-desktop#45]
Is there a way to also pack the database? When I distribute the app to other clients the app works but the database obviously don't.

Let's say I built a very simple todo why when I deliver the app they are not even able to create an account (the account should only be saved in the local database) the app is not supposed to require internet connection to work.

Is it possible somehow?

**Comment by wojtkowiak** [comment]
Hey, sorry for late answer.
This by design a **client only** integration - so it produces a desktop client - just like meteor is building a mobile client for you.
There is a plan to add embedding meteor server and mongo before `1.0` - currently planned for Q1 2017.

more on that:
https://github.com/wojtkowiak/meteor-desktop/issues/39#issuecomment-266134565
https://github.com/wojtkowiak/meteor-desktop/issues/19#issuecomment-260744074

If you would like to help in that, I always warmly welcome any contributions.

**Comment by KoenLav** [comment]
@wojtkowiak do you think this issue falls within the scope of this project?

Meteor already supports local collections: https://guide.meteor.com/collections.html#local-collections

Also there is the GroundDB package: https://github.com/GroundMeteor/db

**Comment by wojtkowiak** [comment]
Yeah, it is not in the scope and was never planned but it seems there is a strong need to have an up to date `Electrify` alternative. Since it can be done as a plugin in `meteor-desktop` I will try to add that but it does not have the highest priority for me.

**Comment by lunafi-evan** [comment]
:+1: this would be a really great ability, even if you need a remote server connection it can still be beneficial to cache things in a local database and get the nice reactive UI updates when storing large amounts of local data
